c2corg_ui/static/js/map/map.js

```diff
@@ -619,6 +625,66 @@ app.MapController.prototype.handleMapSearchChange_ = function(event) {
 
 
 /**
+ * @param {ol.MapBrowserEvent} event
+ * @private
+ */
+app.MapController.prototype.handleMapFeatureClick_ = function(event) {
+  var feature = this.map.forEachFeatureAtPixel(event.pixel, function(feature) {
+    return feature;
+  }, this, function(layer) {
+    // test only features from the current vector layer
+    return layer === this;
```

Comment:
If you are worried about that, you could cache the results of the function before calling `forEachFeatureAtPixel`. But I think that is not a problem.